api.md

```diff
@@ -2722,7 +2721,7 @@ Get information about current license.
 
 |Name|Description|
 |---|---|
-| `edition` | License key type (apache_only, community, enterprise) |
+| `edition` | License type (apache_only, community) |
 | `expired` | Expiration status of license key (bool) |
```

Comment:
I guess expired and expiration_time columns should also be removed here as been done in example below

Reply:
I really don't understand why I haven't removed them. I even thought about it, but forgot. Thank you for noticing it, and sorry, for these left overs! I will fix it right away!

Reply:
done